twilio/types.bal

```diff
@@ -272,3 +272,17 @@ public type MessageResourceResponse record {
     string apiVersion = "";
     json subresourceUris = {};
 };
+
+# Represents voice call message input options
+#
+# + userInput - A Twiml URL or an inline Message that what should be heard when the other party picks up the phone 
+# + userInputType - Whether the userInput is a URL or inline message  
+public type VoiceCallInput record {
+     string userInput;
+     VoiceCallInputType userInputType;
```

Comment:
Shall we take Type first and then the input? Then at Choreo, rendered form will have type first as a combo box and then the input. WDYT?

Reply:
Yes, I changed the order.  :+1: